packages/api/controllers/nomination.js

```diff
@@ -178,15 +178,21 @@ const syncNominations = async (req, res) => {
 const emailVerification = async (req, res) => {
   try {
     const { token } = req.params;
-    const { nomination: id } = jwt.verify(token, process.env.JWT_SECRET);
+    const { data: id } = jwt.verify(token, process.env.JWT_SECRET);
+    console.log(id, '-----nomination------');
     await db.Nomination.update({ emailValidated: true }, { where: { id } });
-    return res.status(200).json({ status: 'ok' });
+    return res.redirect(`${process.env.APP_URL}/api/emailMessage`);
```

Comment:
What do you think about just responding here with the thank you message instead of redirecting to another page?